### DISCUSSION TITLE
Custom yolov8 model with SAHI

### DISCUSSION THREAD

**Discussion by akashsateesha** [obss/sahi#969] [Q&A] [answered]
I have trained a model on visdrone data using pre-trained weights for yolov8 . It is giving a map 0f around 30 . is it a good score ? . Also my next plan of action is to integrate the model I've built with SAHI because most of the objects in visdrone dataset are small . 
how should I go about it ? Also can I implement tracking algorithms for the above integration with SAHI ?

Thank you !!😊

**Comment by ablyzniuk** [comment]
@akashsateesha Have you got any results? I'm struggling with the same task.

**Comment by fcakyon** [comment, marked as answer]
Hey @akashsateesha @ablyzniuk, you can use your custom finetuned yolov8 models with sahi as given here: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov8.ipynb

Moreover, you can follow this example for sahi + yolov8 + object tracking 🚀 https://github.com/ultralytics/ultralytics/pull/8642
